utils/global_events.cfg

```diff
@@ -529,6 +529,9 @@
                 [/not]
             [/filter]
             {UPDATE_STATS}
+            #Make the event undoable
+            {VARIABLE_OP disallow_undo rand(0..1)}
+            {CLEAR_VARIABLE disallow_undo}
```

Comment:
Why is this needed? I don't see how it's related to recalls

Reply:
unit_placed events are always fired on recalls, and before the recall event itself.

Both events do the same thing, [update_stats] if the unit is new and is recalled.

The recall event in practice only serves to clean up starving variable, but I have kept the code for update_stats just in case the unit placed code fails. I can't think of what that case could be, but better to prevent.

Reply:
You should be extremely cautious about adding random calculations to "unit placed" events because in multiplayer "unit placed" events are not synchronized: when a player opens the item menu, "unit placed" for him maybe called many times as he equips many items, all other players will get all the updates in a single pack and run "unit placed" only once. Are you sure this is absolutely necessary here and won't lead to troubles in MP?